src/components/SignInForm.tsx

```diff
@@ -177,7 +179,8 @@ export default function SignIn() {
           <Box sx={{ display: 'flex', flexDirection: 'column', gap: 2 }}>
             <Typography sx={{ textAlign: 'center' }}>
               Don&apos;t have an account?{' '}
-              <Link href="/material-ui/getting-started/templates/sign-in/ody2" sx={{ alignSelf: 'center' }}>
+              <Link component="button"
+              type="button" onClick={() => { void navigate("/signup") }} sx={{ alignSelf: 'center' }}>
```

Comment:
This should also work and is more readable
```suggestion
              <Link component="button"
              type="button" to="/signup" sx={{ alignSelf: 'center' }}>
```